Elevel/Elevel.Api/appsettings.json

```diff
@@ -10,7 +10,7 @@
   "AllowedHosts": "*",
 
   "ConnectionStrings": {
-    "DefaultConnection": "Server=PC\\SQLEXPRESS;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=.\\SQLEXPRESS;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
Please don't add appsettings.json to the commit if you are only changing the string connection.